bashbrew/bashbrew.sh

```diff
@@ -46,6 +46,27 @@ push_image() {
 					fi
 				fi
 			fi
+			if [ "$aliases" ]; then
+				for alias in $aliases; do
+					"$docker" tag "$namespace/$repoTag" "$namespace/$alias:$tag"
+					echo "Pushing alias: $namespace/$alias:$tag..."
+					if ! "$docker" push "$namespace/$alias:$tag" &>> "$thisLog" < /dev/null; then
```

Comment:
It ensures that all I/O streams are not tied to the terminal, which would cause the terminal to hang.